release_notes.md

```diff
@@ -6,6 +6,13 @@ The file describes all the changes, that are affected the usage of the applicati
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
 
+## 0.4.0
+
+### New features
+
+`User profile`: Add the ability for sitters and owners to enter information about themselves. The entered information is saved after the app is closed.
+ 
+`Connect pet sitters and pet owners`: Add a connection to the remote server what enables pet owners to have a list of registered pet sitters.
```

Comment:
```suggestion
`Connect pet sitters and pet owners`: Added the possibility pet owners can see a list of pet sitters and find out additional helpful information as well as the phone number for the selected pet sitter.
```

Reply:
As we discussed, I applied Nadia's version.